src/app/docs/docs.component.html

```diff
@@ -17,4 +17,13 @@
 <app-header>
   <img src="../assets/images/dockstore/dockstore-documentation.png"> Documentation
 </app-header>
-<router-outlet></router-outlet>
+
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="text-center">Dockstore documentation has moved to it's own site!</h2>
+      <p class="text-center">You can find the new docs at the following link:</p>
+      <p class="text-center"><a href="{{redirectLink}}">{{getPrettyLink()}}</a></p>
```

Comment:
Maybe redirect after a few seconds https://stackoverflow.com/questions/3292038/redirect-website-after-certain-amount-of-time